sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/ComplexTypeSuite.scala

```diff
@@ -186,6 +186,37 @@ class ComplexTypeSuite extends SparkFunSuite with ExpressionEvalHelper {
     }
   }
 
+  test("CreateMapFromArray") {
```

Comment:
Could you add more tests with null children and duplicated array items?

Reply:
Tests cases with `null` children: "Array contains", "MapKeys/MapValues" in `CollectionExpressionsSuite.scala` and "CreateArray" in `ComplexTypeSuite.scala`

Reply:
Thank you very much. How about [this test](https://github.com/apache/spark/pull/21258/files#diff-5787751e13a04241f00d846bdca40fdbR220)?